testsuite/integration/picketlink/src/test/java/org/wildfly/test/integration/security/picketlink/core/PartitionManagerProducerTestCase.java

```diff
@@ -71,8 +71,10 @@ public static WebArchive createDeployment() {
                    .addAsManifestResource(new StringAsset("Dependencies: org.picketlink.core meta-inf,org.picketlink.core.api meta-inf, org.picketlink.idm meta-inf\n"), "MANIFEST.MF")
                    .addClass(PartitionManagerProducerTestCase.class)
                     .addAsManifestResource(createPermissionsXmlAsset(
-                            new FilePermission("/tmp/pl-idm", "read,write,delete"),
-                            new FilePermission("/tmp/pl-idm/-", "read,write,delete")),
+                            new FilePermission("/tmp/pl-idm", "read,write,delete"),     // java.io.tmpdir cannot be used here as it has been changed for Arquillian
```

Comment:
The issue here is that the test and the app server the test is running against have java.io.tempdir set to different directories. For the app server the java.io.tempdir points to the system wide tmp directory whileas for the test it points to the working dir of the test. 
If CI has troubles with system wide tmp directory, I can try to reconfigure the app server to have java.io.tempdir pointing to the working dir as well. I'll get back.